core/src/apps/management/change_wipe_code.py

```diff
@@ -56,27 +53,41 @@ async def change_wipe_code(ctx: wire.Context, msg: ChangeWipeCode) -> Success:
     return Success(message=msg_wire)
 
 
-def _require_confirm_action(
+async def _require_confirm_action(
```

Comment:
any particular reason you made this `async` (as opposed to using return type `Awaitable[None]`)?

Reply:
no reason, feel free to change it